autogpt/utils.py

```diff
@@ -11,10 +11,56 @@
 except:
     pass
 
+from autogpt.config import Config
 
-def clean_input(prompt: str = ""):
+cfg = Config()
```

Comment:
Let's pass the plugins by value to the functions instead of keeping this global singleton around.

Reply:
How?

Reply:
You mean as method properties?

Reply:
https://github.com/Significant-Gravitas/Auto-GPT/blob/master/autogpt/agent/agent.py

Reply:
@collijk may also have thoughts about this location

Reply:
Ah now I get it, one second...

Reply:
done